### ISSUE TITLE
Pose Learning

### ISSUE THREAD

**Issue by korizona** [Liumouliu/Deep_blind_PnP#4]
Hello,
Thank you for your contribution with Deep Blind PnP! 
I have been trying to replicate your best model by using the pose loss as shown in section 3.4 of the paper. However i have found some difficulties in producing it. Do you restart the whole training configurations (ex. learning rate) from 1e-3 again or just resume from the model? I have been trying with restarting the learning rate with 1e-3 with no success.

Instead of decreasing, the probability loss keeps increasing. Do i still need to include the original 'BlindPnPLoss' during the second stage of training?

Thank you!
